<?xml version="1.0" encoding="UTF-8"?>

<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>541</height>
   </rect>
  </property>

  <property name="windowTitle">
   <string>자리바꾸기</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <!-- 교탁 -->
   <widget class="QLabel" name="tdesk">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #673A19;background-color: #FDCCBE; border-radius: 7px;</string>
    </property>
    <property name="text">
     <string>교   탁</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>

   <!-- 섞기 버튼 -->
   <widget class="QPushButton" name="btn_rand">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>430</y>
      <width>65</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #66391A; background-color: #FFEFB3; border-style: solid; border-color: #F5DA97; border-radius: 7px; border-width: 3px</string>
    </property>
    <property name="text">
     <string>섞기</string>
    </property>
   </widget>

   <!-- 저장 버튼 -->
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>45</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #66391A; background-color: #FFEFB3; border-style: solid; border-color: #F5DA97; border-radius: 7px; border-width: 3px</string>
    </property>
    <property name="text">
     <string>저장</string>
    </property>
   </widget>

   <!-- 배경 이미지 -->
   <widget class="QLabel" name="label_bg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>BGimage.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>

   <!-- 시작하기 배경 이미지 -->
   <widget class="QLabel" name="intro_bg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>BGimage.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>

   <!-- 시작하기 버튼 -->
   <widget class="QPushButton" name="intro_startbtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #66391A; background-color: #FFEFB3; border-style: solid; border-color: #F5DA97; border-radius: 7px; border-width: 3px</string>
    </property>
    <property name="text">
     <string>시작하기</string>
    </property>
   </widget>

   <!-- 겹침순서 -->
   <zorder>label_bg</zorder>
   <zorder>tdesk</zorder>
   <zorder>btn_rand</zorder>
   <zorder>btn_save</zorder>
   <!-- 아래는 첫 화면이고, 즉시 사라지므로 가장 상위 레이어에 위치 -->
   <zorder>intro_bg</zorder>
   <zorder>intro_startbtn</zorder>
  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
